crates/rspack_plugin_devtool/src/lib.rs

```diff
@@ -100,6 +102,8 @@ impl Plugin for DevtoolPlugin {
     _ctx: PluginContext,
     args: ProcessAssetsArgs<'_>,
   ) -> PluginProcessAssetsOutput {
+    let logger = args.compilation.get_logger(self.name());
```

Comment:
Requiring a compilation for `logger` generation doesn not seem like a good idea. maybe we can
```
let logger = Logger::new(self.name());
....
self.compilation.addLogger(logger);
// or
self.compilation.loggers.push(logger);
```

Reply:
`compilation.addLogger` also require a compilation, `compilation.get_logger` API style is aligned with webpack